sw/device/lib/base/meson.build

```diff
@@ -13,14 +13,42 @@ sw_lib_math_builtins = declare_dependency(
   )
 )
 
+#test('base_math_builtins_unittest', executable(
+#    'base_math_builtins_unittest',
+#    sources: [
+#      'math_builtins_unittest.cc',
+#      'math_builtins.c',
+#    ],
+#    dependencies: [
+#      sw_vendor_gtest,
+#    ],
+#    native: true,
+#  ),
+#  suite: 'base',
+#)
+
```

Comment:
I added the math tests, but I ran into an issue where gtest headers on MacOS include math.h, and for these tests the local math.h overrides the native math.h.  Any ideas how to fix that?

Reply:
any thoughts @mcy?

Reply:
ok